ppu_rules

```diff
@@ -25,6 +25,7 @@ SFO			:=	sfo.py
 
 SPRX			:=	sprxlinker$(POSTFIX)
 CGCOMP			:=	cgcomp$(POSTFIX)
+RAW2H			:=	bin2s$(POSTFIX)
```

Comment:
@miigotu added the original `raw2h` tool back >> https://github.com/ps3dev/PSL1GHT/commit/e573ba2bde3fee8e8cefa0e5b81a15d97c46045c

shouldn't this point to it then?

Reply:
Not if bin2s works already.

Reply:
my concern was about the generated output, just in case the legacy app truly expects `raw2h` output and ends up failing with `bin2s` output. (just a thought)

Reply:
This variable should be added back, with raw2h$ as the target.

Reply:
done in #93